src/main/java/org/mitre/inferno/Validator.java

```diff
@@ -103,12 +103,7 @@ public OperationOutcome validate(byte[] resource, List<String> profiles) throws
    * @return the list of IGs.
    */
   public List<String> getKnownIGs() {
-    try {
-      return getPackageManager().getUrls();
-    } catch (IOException e) {
-      e.printStackTrace();
-      return Arrays.asList("Failed to retrieve Implementation Guides");
-    }
+    return getPackageManager().listPackages();
```

Comment:
This gets changed to `packageManager.listAllIds(igs);` in a future PR which should properly list all the methods now that this method has been deprecated.